src/app/personside/infotabs/meldinger/MeldingerContainer.tsx

```diff
@@ -50,6 +51,13 @@ function MeldingerContainer(props: MeldingerDyplenkeRouteComponentProps) {
     const dyplenker = useInfotabsDyplenker();
     const traadIUrl = useValgtTraadIUrl(props);
     const [sokeord, setSokeord] = useState('');
+    const traaderEtterSok = useSokEtterMeldinger(hasData(traaderResource) ? traaderResource.data : [], sokeord);
+
+    useEffect(() => {
+        if (traadIUrl && traaderEtterSok.length > 0 && !traaderEtterSok.includes(traadIUrl)) {
+            props.history.push(dyplenker.meldinger.link(traaderEtterSok[0]));
```

Comment:
@nutgaard la denne i en egen useEffect. Burde den kanskje legge seg i useEffecten under?

Reply:
fint med en effekt for hver oppgave, hvis du samler alt i en blir det fort kaos. Er også litt fan av å navngi effektene så man kjapt kan se hva som er oppgaven, feks:
```
useEffect( function detteErJobbenMin() {
```
(sier ikke at jeg alltid er flink på det assa :-P)